text/0056-sequence-number-panorama.md

```diff
@@ -0,0 +1,153 @@
+# Sequence number panoramas
+
+## Summary
+
+[summary]: #summary
+
+CEP PR: [casperlabs/ceps#0056](https://github.com/casperlabs/ceps/pull/0056)
+
+We propose citing units by sequence number instead of by hash.
+This will reduce the protocol overhead and bandwidth requirement.
+
+
+## Motivation
+
+[motivation]: #motivation
+
+32-byte hashes of earlier units make up the largest part of Highway messages.
+Replacing them by 8-byte sequence numbers will therefore considerably reduce message sizes,
+and improve performance.
+
+
+## Guide-level explanation
+
+[guide-level-explanation]: #guide-level-explanation
+
+Each unit — the main type of message in the Highway protocol — contains a list of hashes of earlier units.
```

Comment:
```suggestion
Each unit — the main type of message in the Highway protocol — is a hash of hashes of earlier units.
```

Reply:
But a unit is not a hash. It really is a structure _containing_ a panorama, i.e. a list of hashes.

Reply:
My mistake!